### ISSUE TITLE
How to use it on linux platform?

### ISSUE THREAD

**Issue by mildwinter** [UnaNancyOwen/K4ADriver#1]
I have compiled the libk4adriver.so file on the ubuntu16.04 platform and put it in the openni2Driver folder, but I do n’t know how to call it with openni. Can you explain how to call it?

**Comment by UnaNancyOwen** [comment]
(Sorry, I'm not familiar with Ubuntu) I think it just copy libk4adriver.so to <code>OpenNI-Linux-x64-2.2/Redist/OpenNI2/Drivers</code> (and <code>OpenNI-Linux-x64-2.2/Tools/OpenNI2/Drivers</code>).
NiViewer (and this [sample program](https://github.com/UnaNancyOwen/OpenNI2Sample)) will maybe work with Azure Kinect on Ubuntu.

**Comment by mildwinter** [comment]
You are right, copy it and then create a link file of libk4adriver.so.0, openni will automatically try to call the driver file in the Driver folder, thank you for your reply
